### ISSUE TITLE
init_customers yields stripe.error.InvalidRequestError unknown parameter: quantity

### ISSUE THREAD

**Issue by Paul424** [pinax/pinax-stripe#331]
The customers.create function inserts a quantity parameter by default which appears not supported by the Stripe api. It's kind of odd since i could not find any such change/removal in the changelog of the API nor does it work when i configure to use an older API version.

```
stripe.error.InvalidRequestError: Request req_AUqzqofKiCBfeB: Received unknown parameter: quantity
```

```
2017-04-18 10:01:54,630 MainProcess(22912) MainThread(0x7f2078b4c740) stripe INFO util log_info message='Request to Stripe api' method=post path=https://api.stripe.com/v1/customers util.py(100)
2017-04-18 10:01:54,630 MainProcess(22912) MainThread(0x7f2078b4c740) stripe DEBUG util log_debug api_version=2015-10-16 message='Post details' post_data=quantity=1&email=<...> util.py(93)
2017-04-18 10:01:55,381 MainProcess(22912) MainThread(0x7f2078b4c740) stripe INFO util log_info message='Stripe API response' path=https://api.stripe.com/v1/customers response_code=400 util.py(100)
2017-04-18 10:01:55,382 MainProcess(22912) MainThread(0x7f2078b4c740) stripe DEBUG util log_debug body='{\n  "error": {\n    "type": "invalid_request_error",\n    "message": "Received unknown parameter: quantity",\n    "param": "quantity"\n  }\n}\n' message='API response body' util.py(93)
2017-04-18 10:01:55,383 MainProcess(22912) MainThread(0x7f2078b4c740) stripe DEBUG util log_debug link=https://dashboard.stripe.com/test/logs/req_AUqzqofKiCBfeB message='Dashboard link for request' util.py(93)
```

```
Traceback (most recent call last):
  File "manage.py", line 9, in <module>
    execute_from_command_line(sys.argv)
  File "/site/env/python/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 367, in execute_from_command_line
    utility.execute()
  File "/site/env/python/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 359, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/site/env/python/local/lib/python2.7/site-packages/django/core/management/base.py", line 305, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/site/env/python/local/lib/python2.7/site-packages/django/core/management/base.py", line 356, in execute
    output = self.handle(*args, **options)
  File "/site/env/python/local/lib/python2.7/site-packages/pinax/stripe/management/commands/init_customers.py", line 15, in handle
    customers.create(user=user)
  File "/site/env/python/local/lib/python2.7/site-packages/pinax/stripe/actions/customers.py", line 54, in create
    trial_end=trial_end
  File "/site/env/python/local/lib/python2.7/site-packages/stripe/resource.py", line 468, in create
    response, api_key = requestor.request('post', url, params, headers)
  File "/site/env/python/local/lib/python2.7/site-packages/stripe/api_requestor.py", line 142, in request
    resp = self.interpret_response(rbody, rcode, rheaders)
  File "/site/env/python/local/lib/python2.7/site-packages/stripe/api_requestor.py", line 289, in interpret_response
    self.handle_api_error(rbody, rcode, resp, rheaders)
  File "/site/env/python/local/lib/python2.7/site-packages/stripe/api_requestor.py", line 161, in handle_api_error
    rbody, rcode, resp, rheaders)
stripe.error.InvalidRequestError: Request req_AUqzqofKiCBfeB: Received unknown parameter: quantity
```

**Comment by studiorooster** [comment]
Having the same issue...just added question in Pinax Slack channel

**Comment by xfxf** [comment]
I submitted a pull request that solves this, https://github.com/pinax/pinax-stripe/pull/332

**Comment by paltman** [comment]
I believe this one is fixed with #332.
